nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestCommentManagerImpl.java

```diff
@@ -150,6 +151,31 @@ public void testCreateLocalComment() {
         assertThat(commentModel.getPathAsString()).contains(FOLDER_COMMENT_CONTAINER);
     }
 
+    @Test
+    @Override
+    @Ignore("Don't verify this behaviour on this old implementation of comment manager")
+    public void shouldGetCommentedDocModel() {
+
+    }
+
+    @Test
+    @Override
+    @Ignore("Don't verify this behaviour on this old implementation of comment manager")
+    public void shouldNotifyEventWhenCreateComment() {
+    }
+
+    @Test
+    @Override
+    @Ignore("Don't verify this behaviour on this old implementation of comment manager")
+    public void shouldNotifyEventWhenUpdateComment() {
+    }
+
+    @Test
+    @Override
+    @Ignore("Don't verify this behaviour on this old implementation of comment manager")
+    public void shouldNotifyEventWhenRemoveComment() {
+    }
+
```

Comment:
Regarding this, it may be interesting to have a different test stack for the notification tests regarding things to deploy, services handling it etc.

Reply:
yes its why i said you yesterday about having a dedicated test of notification to avoir started a smtp server (on master version), but if we do it i think we should create three class tests + one as abstract to covers all the comment managers implementation. i don't which is the best approche ;)

Reply:
I would rather having more test classes representing real thing to test than having to disable tests in cases it doesn't fit.

We can ask the team about this.